package.json

```diff
@@ -3,10 +3,16 @@
   "description": "Jsonnet libraries to help with GoCD",
   "type": "module",
   "scripts": {
-    "test": "ava"
+    "test": "ava",
+    "eslint": "eslint --config './.eslintrc.json' .",
```

Comment:
That's the default value for --config, so I'm not sure this should be here.

https://eslint.org/docs/latest/use/configure/configuration-files#configuration-file-formats

P4: Not a blocking issue, at all.